### ISSUE TITLE
ERROR iron::response - Error writing response

### ISSUE THREAD

**Issue by Boscop** [iron/iron#518]
I wanted to test my app on mobile, so I ran the server on my laptop and tested it on my phone, over the wifi router, but often the page didn't load completely, and the server logged the following:
```
ERROR iron::response - Error writing response: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. (os error 10060)
```
(It's running on Windows 8.1 btw).
This issue never happens when I access the site from the same laptop.
My phone is a Oneplus One and the browser is Chrome Mobile, it never has such problems accessing other websites.
What could be going wrong?
Is it the fault of the wifi router?

**Comment by untitaker** [comment]
You're most likely listening on `127.0.0.1` when you have to listen on `0.0.0.0`.

**Comment by Boscop** [comment]
No, I'm listening on 0.0.0.0:3000:
```ini
FULL_URL=http://192.168.178.35:3000
LISTEN_AT=0.0.0.0:3000
```
The FULL_URL is so that redirects work on the phone..

**Comment by untitaker** [comment]
Then it is perhaps a firewall from Windows?

**Comment by Boscop** [comment]
No, I had that problem before, but then I allowed the exe on both private and public networks.
Before that I got `ERR_CONNECTION_TIMED_OUT` on the phone (refused to connect), after that I got the content, but in some cases it fails and only loads the site to ~75% before this error happens. Then I refresh and it succeeds. I definitely think it's not a firewall issue.

**Comment by untitaker** [comment]
Do you have a comparable experience with any other development server? Such as
`python3 -mhttp.server`.

**Comment by Boscop** [comment]
This project can't be served with a simple file server because it's using maud to render the html and needs db access etc. 
But I tested another project (the Polymer starter kit) with `python -m SimpleHTTPServer` (I don't have Python 3 installed.) and when I access it from my phone through the wifi router, I noticed it loads very slowly, although it still finishes getting the whole page in the end.
Maybe there is a cutoff point, if it would have taken slightly longer, I would see the same result?
But why would it take so long to load it via LAN, the wifi router is a modern Fritzbox router.

**Comment by untitaker** [comment]
>This project can't be served with a simple file server because it's using maud to render the html and needs db access etc.

Sure, but I meant to test whether this is a network problem. So which project you choose wouldn't matter. Python's builtin server is very slow, that is known.

I don't know what's wrong with it. Could you check whether this happens with other Hyper-based frameworks, or whether this problem occurs with clients other than the laptop it's running on and your smartphone?
